public/js/components/Scopes.js

```diff
@@ -71,15 +73,99 @@ function getScopes(pauseInfo) {
       scopes.push([
         scope.getIn(["object", "class"]),
         fromJS({
+          isScope: true,
           value: scope.get("object")
         })
       ]);
     }
+    index++;
   } while (scope = scope.get("parent")); // eslint-disable-line no-cond-assign
 
   return scopes;
 }
 
+function isObject(obj) {
+  return obj.getIn(["value", "type"]) === "object";
+}
+
+function loadedProperties(obj, loadedObjects) {
+  return loadedObjects.get(obj.getIn(["value", "actor"])) ||
+         obj.get("prefetchedProperties");
+}
+
+function sortProperties(a, b) {
+  const obj = isObject(a[1]) && !isObject(b[1]) ? -10 : 10;
+  const alpha = a[0] < b[0] ? -1 : 1
+
+  return obj + alpha > 0 ? -1 : 1;
```

Comment:
Sounds good. I felt bad about breaking the alphabetical sorting. Merge and improve